double-check-tmp.md

```diff
@@ -0,0 +1,28 @@
+- No copyright/license mentioned
+  + http://www.jjj.de/fxt/fxtbook.pdf
```

Comment:
License is at the end: Creative Commons Attribution-Noncommercial-No Derivative Works 3.0.  Good.